desktop/apps/categories3/components/FeaturedGenes.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+import FeaturedGene from './FeaturedGene'
+
+const propTypes = {
+  featuredGenes: PropTypes.object
+}
+
+const Layout = styled.div`
+  padding-top: 1em;
+  column-count: 3;
+  column-gap: 2em;
```

Comment:
We are not trying to make number of columns depend on a screen width here?

Reply:
Ah good point. Might also make sense to simply suppress this at small screen sizes.